test/node-test.js

```diff
@@ -16,12 +16,23 @@ const Peer = require('../lib/net/peer');
 const InvItem = require('../lib/primitives/invitem');
 const invTypes = InvItem.types;
 
+const ports = {
+  p2p: 49331,
```

Comment:
Should we document that these ports will be used when running the tests?

Reply:
We could, however I think we would need a `docs/testing.md` first, that can probably be added sometime soon, especially when adding fuzzing, benchmarking, integration, and e2e tests suites.

Reply:
See this PR in hsd for a first pass at a `test/README.md`
https://github.com/handshake-org/hsd/pull/115/files#diff-c30ff793110b0960f76bb3854af6f5a8

@boymanjor and I had a meeting and decided on this particular way of setting up the tests. Any feedback would be greatly appreciated, and then we can port it over to bcoin